src/sequence_state.cc

```diff
@@ -300,7 +370,8 @@ SequenceStates::CopyAsNull(const std::shared_ptr<SequenceStates>& from)
           std::forward_as_tuple(from_input_state_tensor->Name()),
           std::forward_as_tuple(new SequenceState(
               from_input_state_tensor->Name(), from_input_state_tensor->DType(),
-              from_input_state_tensor->Shape())));
+              from_input_state_tensor->Shape(), false /* reused buffer */,
+              false /* use growable memory */)));
```

Comment:
nit: match var names more closely for Ctrl-F/grep hits when looking through the code
```suggestion
              from_input_state_tensor->Shape(), false /* reuse_buffer */,
              false /* use_growable_memory */)));
```

Reply:
fixed